test-data/unit/check-python311.test

```diff
@@ -196,9 +196,38 @@ def foo():
                 return  # E: "return" not allowed in except* block
 [builtins fixtures/exception.pyi]
 
+[case testContinueInExceptBlockNestedInExceptStarBlock]
+# flags: --python-version 3.11
+while True:
+    try:
+        ...
+    except* Exception:
+        try:
+            ...
+        except Exception:
+            continue  # E: "continue" not allowed in except* block
+        continue  # E: "continue" not allowed in except* block
+[builtins fixtures/exception.pyi]
+
+[case testReturnInExceptBlockNestedInExceptStarBlock]
+# flags: --python-version 3.11
+def foo():
+    try:
+        ...
+    except* Exception:
+        try:
+            ...
+        except Exception:
+            return  # E: "return" not allowed in except* block
+        return  # E: "return" not allowed in except* block
+[builtins fixtures/exception.pyi]
+
 [case testBreakContinueReturnInExceptStarBlock1]
 # flags: --python-version 3.11
-class range: pass
+from typing import Sequence
+class range(Sequence[int]): 
+    def __init__(self, __x: int, __y: int = ..., __z: int = ...) -> None: pass
```

Comment:
Can this be replaced by a dummy list?

Reply:
So basically replace `range(5)` with `list` in the `for` loops?